packages/custom_lint_core/lib/src/configs.dart

```diff
@@ -108,11 +110,33 @@ class CustomLintConfigs {
       }
     }
 
+    final errors = <String, ErrorSeverity>{...includedOptions.errors};
+
+    final errorsYaml = customLint['errors'] as Object?;
+    if (errorsYaml is Map) {
+      for (final entry in errorsYaml.entries) {
+        final value = entry.value;
+        if (entry.key case final String key?) {
+          final severity = ErrorSeverity.values.firstWhereOrNull(
+            (e) => e.displayName == value,
```

Comment:
Let's not rely on "displayName" .
There is no guarantee about its stability. This could indirectly break custom_lint

That has already happened to me before, so I won't fall for it twice :P

Reply:
Instead we can do a simple switch:

```dart
ErrorSeverity parseSeverity(String str) {
  return switch (str) {
    'error' => ErrorSeverity.error,
    'ignore' => ErrorSeverity.none,
     ...,
     _ => throw UnsupportedError('Unsupported severity $str'),
  };
}
```